src/database/migrations/20231211041333_CreateApiTable.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex';
+
+import { ApiRole } from '../../api/entities/ApiRole';
```

Comment:
I think a lot of these imports are unnecessary - better to remove them if they're not needed.